### ISSUE TITLE
tuckr ignores directory `Configs/foo_macos` if `Configs/foo` does not exist.

### ISSUE THREAD

**Issue by jasha-hrp** [RaphGL/Tuckr#33]
Tuckr ignores a folder with a os suffix (e.g. `foo_macos`) if the corresponding non-suffixed folder `foo` does not exist.

For example, if my .dotfiles directory looks like this:
```
      - │           ┌─ yazi.toml
      - │        ┌─ yazi
      - │     ┌─ .config
      - │  ┌─ yazi_macos
      - ├─ Configs
57344 B .dotfiles
```
then runing `tuckr add yazi` fails:
```
$ tuckr add yazi                                  
yazi doesn't exist.
```
I was able to make `tuckr add yazi` work by creating a `yazi` folder next to the `yazi_macos` folder:
```
      - │           ┌─ yazi.toml
      - │        ┌─ yazi
      - │     ┌─ .config
      - │  ┌─ yazi_macos
      - │  │  ┌─ .gitkeep
      - │  ├─ yazi
      - ├─ Configs
57344 B .dotfiles
```
I created a file `Configs/yazi/.gitkeep` file so that the `yazi` folder could be checked into version control. As a result, `tuckr` creates a symlink to the `.gitkeep` file in my home directory, which I'd like to avoid.

Would you be willing to consider one of the following modifications to tuckr?
- Have `tuckr add foo` add files from e.g. `Configs/foo_macos` even if the folder `Configs/foo` does not exist
- Have `tuckr` ignore files titled `.gitkeep` (or e.g. files titled `.tuckrignore`)

**Comment by RaphGL** [comment]
I prefer adding the first option where the base group `foo` is not needed. It'll probably require some change in the way I'm handling groups but we'll see how it goes :)

**Comment by RaphGL** [comment]
So it's been a while since this issue was reported. Back when this was reported there was no way of actually achieving this properly as I recall, since I tried a few times and it just needed too much to be able to do it properly. 

But today, I decided to give it a shot again and it now seems to be "easily" achievable. 
I managed to implement this functionality and make it work, but while attempting to make a unit test work I ended up losing all my progress :))))

This was actually going to ship into the 0.10 release but since I wanted to get the latest version on people's hands asap, I've marked this as 0.11. But since this is actually a usability issue for some, I might end up making a minor release with this fix along with the secrets security patch.
